test/rfc7540_SUITE.erl

```diff
@@ -375,6 +375,36 @@ lingering_data_counts_toward_connection_window(_) ->
 	timer:sleep(300),
 	gun:close(ConnPid).
 
+respect_max_concurrent_streams(_) ->
+	doc("The SETTINGS_MAX_CONCURRENT_STREAMS setting can be used to "
+		"restrict the number of concurrent streams. (RFC7540 5.1.2, RFC7540 6.5.2)"),
+	{ok, Ref, Port} = do_cowboy_max_councurrent_streams(1),
+	try
+		{ok, ConnPid} = gun:open("localhost", Port, #{protocols => [http2]}),
+		{ok, http2} = gun:await_up(ConnPid),
+		StreamRef1 = gun:get(ConnPid, "/"),
+		timer:sleep(100),
```

Comment:
Probably don't need the timer:sleep here.

Reply:
Done.